docs/application/web/get-started/web-service/first-service.md

```diff
@@ -0,0 +1,223 @@
+# Create Your First Web Service Application
+
+**Welcome to Web service application development!**
+
+A Web service consists of JavaScript file with its business logic running on the Node.js runtime and a config.xml file specifying a Tizen package.
+This means that you can also use Tizen Web service life-cycle and Node.js built-in modules.
```

Comment:
Is there any reason add the second sentence to first? I think the original is not bad. Doesn't the suggestion make sentence too long? What is the benefit you want to get with merging the 2 sentences into one?

Reply:
@mijin85cho  thanks for your question, 

I joined the sentence because of "this" in this case refers back to the context and the meaning of previous sentence. If you say, we can detach to original.

Reply:
Done